Bug in the type definitions

Hi, if I start using your library in an typescript application, I get the following error:
```
ERROR in node_modules/ical-generator/index.d.ts(54,14): error TS2709: Cannot use namespace 'moment' as a type.
```

In https://github.com/sebbo2002/ical-generator/blob/develop/index.d.ts#L54 your are using the `moment` namespace as a type, I think it should be `moment.Moment | Date` there, just like the `end` property.

--- a/index.d.ts
+++ b/index.d.ts
@@ -51,7 +51,7 @@ declare module 'ical-generator' {
      * Information used to create calendar events
      */
     interface EventData {
-      start: moment;
+      start: moment.Moment | Date;
       summary: string;
       id?: string;
       uid?: string;

